GoogleDataTransport/GDTLibrary/GDTUploadCoordinator.m

```diff
@@ -208,32 +208,54 @@ - (void)appWillTerminate:(GDTApplication *)application {
 #pragma mark - GDTUploadPackageProtocol
 
 - (void)packageDelivered:(GDTUploadPackage *)package successful:(BOOL)successful {
-  dispatch_async(_coordinationQueue, ^{
+  dispatch_queue_t coordinationQueue = _coordinationQueue;
+  if (!_coordinationQueue) {
```

Comment:
I don't understand this one - since it's synchronous in an instance method

Reply:
it could probably be replaced with just a check to see if non-nil.

Reply:
the check here was also wrong.